src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -391,6 +391,10 @@ var VALID_TAG_REGEX = /^[a-zA-Z][a-zA-Z:_\.\-\d]*$/; // Simplified subset
 var validatedTagCache = {};
 var hasOwnProperty = {}.hasOwnProperty;
 
+// In IE8 We dont have HTMLUnknownElement - thus we use the equivalent HTMLGenericElement
+var HTMLUnknownElement = typeof window.HTMLGenericElement !== 'undefined' ?
```

Comment:
I don't believe React actually supports IE8 in DEV anymore so this should be redundant then. Devs?